src/gui/tray/ActivityItem.qml

```diff
@@ -139,6 +145,31 @@ MouseArea {
                 }
                 
             }
+
+            Button {
+                id: shareButton
+                
+                Layout.preferredWidth: (path === "") ? 0 : parent.height
```

Comment:
is the condition really needed ?
I mean you already set visible to false so it will get ignored by the layout if `visible == false`

Reply:
Yes, that is really needed.

Reply:
the condition is redundant as the `Button` item will never be visible when `path !== ""` so it will not take place in the layout

Reply:
see example
```
import QtQml 2.15
import QtQuick 2.15
import QtQuick.Controls 2.15
import QtQuick.Layouts 1.15

RowLayout {
    spacing: 0
    Rectangle {
        color: 'red'
        Layout.fillHeight: true
        Layout.preferredWidth: 50
    }

    Rectangle {
        color: 'green'
        visible: false
        Layout.fillHeight: true
        Layout.preferredWidth: 50
    }

    Rectangle {
        color: 'blue'
        Layout.fillHeight: true
        Layout.preferredWidth: 50
    }
}
```

Reply:
mh, I see that it works now. I wonder what my problem was.